cypress/support/config.ts

```diff
@@ -45,6 +45,7 @@ beforeEach(() => {
         termsOfUse: 'https://example.com/termsOfUse',
         imprint: 'https://example.com/imprint'
       },
+      plantumlServer: 'http://www.plantuml.com/plantuml',
```

Comment:
Please don't use a live plantuml server for the test. It slows down the test runner, requires access to the internet and the test could fail if plantuml is down. How about a `cy.intercept` mock?

Reply:
How do you want to mock PlantUML? Returning a predefined "diagram" image url?

Reply:
Yes

Reply:
Done